### DISCUSSION TITLE
`--` is not supported

### DISCUSSION THREAD

**Discussion by damanis** [linrongbin16/fzfx.nvim#745] [Q&A]
Before search line after `--` accepted additional arguments for `rg`, e.g. `... -- -t js` -- search only in JavaScript files.
Is this feature removed,

**Comment by linrongbin16** [comment]
No, they should works.
Can you provide some command lines? I could take a look.

**Comment by damanis** [comment]
It is not stable, I did't catch 100% scenario.
Today, for example, `some_text -- -t js` works properly, while `some_text -- -T js` find few files. Forked `rg` command looks properly; also in shell `rg` with `-t` and `-T` works as expected.

**Reply by linrongbin16** [reply]
let me try
